Firestore/Example/FuzzTests/FSTFuzzTestsPrincipal.mm

```diff
@@ -18,16 +18,23 @@
 
 #include "LibFuzzer/FuzzerDefs.h"
 
+#include "Firestore/core/src/firebase/firestore/model/database_id.h"
 #include "Firestore/core/src/firebase/firestore/remote/serializer.h"
 
+using firebase::firestore::model::DatabaseId;
 using firebase::firestore::remote::Serializer;
 
 namespace {
 
 // Fuzz-test the deserialization process in Firestore. The Serializer reads raw
 // bytes and converts them to a model object.
 void FuzzTestDeserialization(const uint8_t *data, size_t size) {
-  // TODO(minafarid): fuzz-test Serializer.
+  DatabaseId database_id{"project", DatabaseId::kDefault};
+  Serializer serializer(database_id);
```

Comment:
Nit: `database_id` and `serializer` use different, largely equivalent syntaxes to call the constructor (`()` vs `{}` braces). I'd suggest using one for consistency (probably the newer "universal" initialization syntax `{}`, although I don't feel strongly about this).

Reply:
Ok, modified it to use `{}`